cmd/keycmd/delete.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package keycmd
+
+import (
+	"errors"
+	"os"
+
+	"github.com/ava-labs/avalanche-cli/pkg/prompts"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/spf13/cobra"
+)
+
+var forceDelete bool
```

Comment:
maybe can just be called force. just a nit I know

Reply:
the reason I did this is there is also a `--force` flag for the `create` command. I think it's better to disambiguate, so I'm going to leave it as is for now, but could be convinced to consolidate flag variables later.